internal/cleanup/config.go

```diff
@@ -40,6 +42,8 @@ func (c *Config) DB() *database.Config {
 }
 
 // BlobStorage returns the BlobStorage configuration.
-func (c *Config) BlobStorage() bool {
-	return true
+func (c *Config) BlobStorage() storage.BlobstoreConfig {
+	return storage.BlobstoreConfig{
```

Comment:
this same function over and over again seems like an anti-pattern.

Why not have BlobStorage return a string and centralize the creation in Setup.

The whole "factory" concept can possible be removed if this is only done in one place.

Reply:
I changed this as you suggested to keep at simple. The initial idea was to keep it more extensible, because different apps that use the same Setup might want to use different strategies for Blobstore creation.